target/linux/ath79/image/generic.mk

```diff
@@ -171,6 +171,15 @@ define Device/comfast_cf-e120a-v3
 endef
 TARGET_DEVICES += comfast_cf-e120a-v3
 
+define Device/comfast_cf-e314n-v2
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-E314N
+  DEVICE_VARIANT := v2
+  DEVICE_PACKAGES := rssileds kmod-leds-gpio kmod-leds-reset kmod-i2c-gpio
```

Comment:
`kmod-leds-gpio kmod-leds-reset kmod-i2c-gpio` should all not be necessary here.

Reply:
but kmod-leds-gpio yes, right ?

Reply:
It's compiled into the kernel, so you do not need to add the module. See https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/config-4.14#L134

Reply:
ok done